README.md

```diff
@@ -844,6 +844,27 @@ create function get_random_number() returns int as $$
 $$ language sql stable;
 ```
 
+## Running Graphile Migrate from a container
+
+When working in a team it can be useful to package `graphile-migrate` and the
+postgres tooling into a docker file so that everyone has easy access to the
+same versions. This is also helpful if you're using Migrate as part of a larger
+non-nodejs based project.
+
+For these purposes we provide an example docker file in `scripts/Dockerfile`.
+This uses the latest released version of graphile-migrate from `npm` and
+packages it together with the necessary nodejs and postgres tools. You can
+build the docker file from the root of this repository using a command like
+
+```bash
+docker build -t graphile-migrate scripts \
+    --build-arg NODEJS_VERSION=14 --build-arg POSTGRES_VERSION=12
+```
+
+To conveniently run Graphile Migrate within the container you can then use the
+wrapper script in `scripts/graphile-migrate` which passes the standard Migrate
+environment variables through to the container.
+
```

Comment:
Let's make this a link to the relevant docs and move the content into `docs/docker/README.md`

```suggestion
## Examples

- [Running Graphile Migrate in a Docker container](docs/docker/README.md)
- [Examples of idempotent migration files including edge cases](docs/idempotent-examples.md)
```